lib/ansible/playbook/role/__init__.py

```diff
@@ -358,6 +358,7 @@ def _create_validation_task(self, argument_spec, entrypoint_name):
                 },
             },
             'name': task_name,
+            'tags': ['always'],
```

Comment:
We should probably document this behavior similarly to how we document how it affects fact gathering:

https://github.com/ansible/ansible/blob/80e7e1a17cb6aaf10e289e671fc382a614018e6e/docs/docsite/rst/user_guide/playbooks_tags.rst#L304-L306

Otherwise I can see someone being surprised that role arg spec validation is skipped when they run `--tags mytag --skip-tags always` if they want to skip tasks they have tagged with `always` in their play.

Reply:
Great point.